lib/__tests__/standalone.test.mjs

```diff
@@ -287,6 +287,66 @@ it('unknown formatter option', async () => {
 	).rejects.toThrow(/^You must use a valid formatter option/);
 });
 
+describe('mergeRule', () => {
+	it('ignores configFile when config is passed and mergeRules is disabled (code)', async () => {
```

Comment:
Since `mergeRules` default is `false` this test and l305 _should_ have probably already be covered elsewhere, but I wanted to prevent regressions so I tested the status quo.